lib/gui/pages/finish/templates/success.tpl.html

```diff
@@ -15,20 +15,20 @@ <h3 class="title"><span class="tick tick--success space-right-tiny"></span> Flas
     <div class="box center">
       <div class="fallback-banner">
         <div class="caption caption-big">Thanks for using
-          <svg-icon path="../../../assets/etcher.svg"
-            width="150px"
-            height="auto"
+          <svg-icon path="'../assets/etcher.svg'"
```

Comment:
It's relative to the SVGIcon module which did move, it's a bit of a mess but it should be resolved once we start using JSX and transpiling the source into a single bundled file and everything is relative to that, unless there are any more immediate fixes we could apply. It has bit me recently in the notification issue, where there is no error emitted at wrongful paths.